spikeinterface/preprocessing/tests/test_highpass_spatial_filter.py

```diff
@@ -0,0 +1,190 @@
+import spikeinterface as si
+import spikeinterface.preprocessing as spre
+import spikeinterface.extractors as se
+import pytest
+import numpy as np
+from copy import deepcopy
+
+from spikeinterface.core.testing_tools import generate_recording
+import spikeinterface.widgets as sw
+
+try:
+    import spikeglx
+    import neurodsp.voltage as voltage
+    HAVE_IBL_NPIX = True
+except ImportError:
+    HAVE_IBL_NPIX = False
```

Comment:
Testing against IBL should be done locally only and not on github.
I propose to make local import in the function

Reply:
yep, removing